test/testutil/buffer_generator.hpp

```diff
@@ -0,0 +1,40 @@
+/**
```

Comment:
Randomness in test may be difficult to debug and change behavior.

Remove `BufferGenerator` and `CidGenerator`.
Use pregenerated inputs with `"..."_unhex` and `ContentIdentifierCodec::decode("..."_unhex).value`.
May create some global constants, like `kCid1`, `kCid2`